examples/long_tutorials/handle_drift/plot_handle_drift.py

```diff
@@ -0,0 +1,342 @@
+#   * **bottom left:** The average motion vector across depths and all motion across spatial depths (for non-rigid estimation).
+#   * **bottom right:** if motion correction is non rigid, the motion vector across depths is plotted as a map, with the color code representing the motion in micrometers.
+
+for preset in presets_to_run:
+
+    fig = plt.figure(figsize=(7, 7))
+
+    si.plot_motion_info(
+        results[preset]["motion_info"],
+        recording=corrected_recording,  # the recording is only used to get the real times
+        figure=fig,
+        depth_lim=(400, 600),
+        color_amplitude=True,
+        amplitude_cmap="inferno",
+        scatter_decimate=10,            # Only plot every 10th peak
+    )
+    fig.suptitle(f"{preset=}")
+
+# %%
+# These plots are quite complicated, so it is worth covering them in detail.
+# For every detected spike in our recording, we first estimate
+# its depth (first panel) using a method from
+# :py:func:`~spikeinterface.postprocessing.compute_unit_locations()`.
+#
+# Then, the probe motion is estimated the location of the detected peaks are
+# adjusted to account for this (second panel).
+#
+# The motion estimation produces
+# a measure of how much and in what direction the probe is moving at any given
+# time bin (third panel). For non-rigid motion correction, the probe is divided
+# into subsections - the motion vectors displayed are per subsection (i.e. per
+# 'binned spatial depth') as well as the average.
+#
+# On the fourth panel, we see a
+# more detailed representation of the motion vectors. We can see the motion plotted
+# as a heatmap at each binned spatial depth across all time bins. It captures
+# the zigzag pattern (alternating light and dark colors) of the injected motion.
+
+# %%
+# A few comments on the figures:
+#   * The preset **'rigid_fast'** has only one motion vector for the entire probe because it is a 'rigid' case.
+#     The motion amplitude is globally underestimated because it averages across depths.
+#     However, the corrected peaks are flatter than the non-corrected ones, so the job is partially done.
+#     The big jump at=600s when the probe start moving is recovered quite well.
+#   * The preset **kilosort_like** gives better results because it is a non-rigid case.
+#     The motion vector is computed for different depths.
+#     The corrected peak locations are flatter than the rigid case.
+#     The motion vector map is still be a bit noisy at some depths (e.g around 1000um).
+#   * The preset **nonrigid_accurate** seems to give the best results on this recording.
+#     The motion vector seems less noisy globally, but it is not 'perfect' (see at the top of the probe 3200um to 3800um).
+#     Also note that in the first part of the recording before the imposed motion (0-600s) we clearly have a non-rigid motion:
+#     the upper part of the probe (2000-3000um) experience some drifts, but the lower part (0-1000um) is relatively stable.
```

Comment:
I don't like "drifts" since I would say this is an uncountable concept, but it may be too late and the field has settled on this.

Reply:
Yes I wanted to fix to 'motion', but its had to use in some senses e.g. 'upper part of the probe may experience some motion' doesn't quite read properly for me. I guess motion may be too general and makes you think that probe is moving (or maybe the probe is moving because the animal is moving! there is 'motion' in the system) whereas 'drift' is a bit more specific and particularly refers to the phenomenon of the signal drifting across the probe (maybe the probe or may be the brain).